app/views/people/_form.html.erb

```diff
@@ -57,8 +60,9 @@
     <%= form.label :user_type %>
     <%= form.text_field :user_type, id: :person_user_type %>
   </div>
+  <% end %>
```

Comment:
Delete this entire block 53-63. Should not be able to change user id or user type.